src/node/desktop/envvars.bat

```diff
@@ -4,6 +4,7 @@ REM Run this script with 'call envvars.bat' to update the PATH in your
 REM cmd.exe shell, so that the appropriate versions of node are found
 set NODE_PATHS=^
 	..\..\..\dependencies\common\node\16.20.2^
+	..\..\dependencies\common\node\16.20.2^
```

Comment:
Why is this here? seems like the dependencies folder 3 levels up is correct

Reply:
Because I needed to source this from the `src/gwt` folder, because the build tasks were failing to find node. So really ... just a way to be lazy.